lib/MusicBrainz/Server/Entity/URL/ISRCTW.pm

```diff
@@ -5,11 +5,7 @@ use Moose;
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
-sub sidebar_name {
-    my $self = shift;
-
-    return "ISRC\N{U+67E5}\N{U+8A62}\N{U+7CFB}\N{U+7D71}";
-}
+sub sidebar_name { return "ISRC\N{U+67E5}\N{U+8A62}\N{U+7CFB}\N{U+7D71}" }
```

Comment:
(Not that it matters, but since the goal of the PR was simplifying everything) you forgot to remove the `return` here. :)